### PR TITLE
Added Florian's talk to new branch

### PR DESCRIPTION
**Pull request by emmasofiah2o**
- Added file
- Added picture and logo

### CODE REVIEW

**Comment by emmasofiah2o** [comment]
@marcoow moved this to a new branch since I had commits to the old branch as well (and not only the pr). It is failing tests tho now...

**Comment by marcoow** [comment]
this breaks the build:

```
ENOENT: no such file or directory, open '/home/runner/work/mainmatter.com/mainmatter.com/static/assets/images/talks/2022-12-09-embereurope-2022/properly-flaky-modifiers-thumbnail.jpg' (via Template render error)
```

**Comment by emmasofiah2o** [comment]
@marcoow all fixed, can I merge?

File: src/appearances/2022-12-09-properly-flaky-modifiers.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Properly flaky modifiers"
+image: "/assets/images/talks/2022-12-09-embereurope-2022/properly-flaky-modifiers.jpg"
+url: https://www.youtube.com/watch?v=3NgStUiki7k
+media: video
+channel: embereurope-2022
+---
+
+Florian Pichler showed how to build a snowflake effect with modifiers and Ember
+components the last Ember Europe Meetup of 2022.
```
Comment (marcoow) [inline src/appearances/2022-12-09-properly-flaky-modifiers.md]:
```suggestion
components at the last Ember Europe Q4 2022 Meetup.
```

File: src/appearances/2022-12-09-properly-flaky-modifiers.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Properly flaky modifiers"
+image: "/assets/images/talks/2022-12-09-embereurope-2022/properly-flaky-modifiers.jpg"
+url: https://www.youtube.com/watch?v=3NgStUiki7k
+media: video
+channel: embereurope-2022
+---
+
+Florian Pichler showed how to build a snowflake effect with modifiers and Ember
+components at the last Ember Europe Q4 2022 Meetup.
```
Comment (marcoow) [inline src/appearances/2022-12-09-properly-flaky-modifiers.md:10]: @emmasofiah2o just realize this is still wrong – there's only one Ember Europe Q4 of course so no last one, please remove "last"

**Pull request merged**